src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -168,7 +190,7 @@ impl LeaderBlockCommitOp {
         let parent_vtxindex = parse_u16_from_be(&data[68..70]).unwrap();
         let key_block_ptr = parse_u32_from_be(&data[70..74]).unwrap();
         let key_vtxindex = parse_u16_from_be(&data[74..76]).unwrap();
-        let memo = data[76..77].to_vec();
+        let burn_parent_modulus = ((data[76] as u64) % BURN_BLOCK_MINED_AT_MODULUS) as u8;
```

Comment:
Can you add a bitmask here so that the burn parent modulus only comes from the lowest 3 bits?  This lets use use the remaining 5 bits as a memo field (which may be required for addressing #1805).

Reply:
Sure

Reply:
Added in 5972a533a